libs/designsystem/src/lib/components/item/item.component.spec.ts

```diff
@@ -1,24 +1,79 @@
-import { Spectator, createComponentFactory } from '@ngneat/spectator';
-import { MockModule } from 'ng-mocks';
 import { IonicModule } from '@ionic/angular';
+import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
+import { MockModule } from 'ng-mocks';
 
+import { DesignTokenHelper } from '../../helpers/design-token-helper';
 import { IconModule } from '../icon/icon.module';
+
 import { ItemComponent } from './item.component';
 
+const itemHeight = DesignTokenHelper.itemHeight;
+const size = DesignTokenHelper.size;
```

Comment:
Consider simplifying;
```suggestion
const { itemHeight, size } = DesignTokenHelper;
```